pages/dev/u/_username.vue

```diff
@@ -1,24 +1,89 @@
 <template lang="html">
-  <div class="screen">
+  <div class="screen background-tint">
```

Comment:
does this still need to be included if we change the main.css background selector from 'body' to 'html'?

Reply:
almost all of those background tints are left over from when I didn't port most of the page styling to global main.css.  Not needed, but not hurting right now. I'll get them in my next branch